stan/stan.go

```diff
@@ -11,6 +11,13 @@
 //   client_cert, client_key
 //     nats client certificate and key file paths.
 //
+// bps.NewSubscriber supports:
+//
+//   queue_group
+//     optional queue group name for queue subscriptions: https://docs.nats.io/developing-with-nats/receiving/queues
+//   durable_name
+//     durable subscriptions name: https://docs.nats.io/developing-with-nats-streaming/durables
```

Comment:
These are actually subscription (topic) level things, but IMO it's totally OK to have them on client level: I think the main BPS usage pattern is a single client with the same settings for all the topics/subs.